### PR TITLE
DM-1464 : AS ID, AS Key, admin and core API version along with the screenshot

### PR DESCRIPTION
**Pull request by atulpandasag**
…creenshot.

### CODE REVIEW

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 1]
```diff
@@ -19,8 +19,10 @@ Enter the following information:
...
-- **Route Application Server ID**: TLS security ID between provider platform and agent. The application server ID should be unique to the route address. This is an optional field. Leave empty to disable security.
-- **Route Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change the application server key if  the application server ID is not updated. This is an optional field. Leave empty to disable security.
+- **Application Server ID**: TLS security ID between provider platform and agent. The application server ID should be unique to the route address. This is an optional field. Leave empty to disable security.
+- **Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change the application server key if  the application server ID is not updated. This is an optional field. Leave empty to disable security.
+- **Admin API version**: The version that the Thingpark admin API uses. By-default it will be set to "latest".
+- **Core API version**:  The version that the Thingpark core API uses. By-default it will be set to "latest".
```
Comment (rssag) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:25]:
```suggestion
- **Admin API version**: The version that the Thingpark admin API uses. The default is "latest".
- **Core API version**:  The version that the Thingpark core API uses. The default is "latest".
```

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
Which releases does this apply to?

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 2]
```diff
@@ -19,8 +19,10 @@ Enter the following information:
...
 - **Profile ID**: This depends on your ThingPark account and environment. If you are using, for example, the Dev1 ThingPark environment your profile ID will be "dev1-api". Multiple tenants can have the same profile ID.
-- **Route Application Server ID**: TLS security ID between provider platform and agent. The application server ID should be unique to the route address. This is an optional field. Leave empty to disable security.
-- **Route Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change the application server key if  the application server ID is not updated. This is an optional field. Leave empty to disable security.
+- **Application Server ID**: TLS security ID between provider platform and agent. The application server ID should be unique to the route address. This is an optional field. Leave empty to disable security.
+- **Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change the application server key if  the application server ID is not updated. This is an optional field. Leave empty to disable security.
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:23]:
```suggestion
- **Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change the application server key if the application server ID is not updated. This is an optional field. Leave empty to disable security.
```

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 3]
```diff
@@ -19,8 +19,10 @@ Enter the following information:
...
-- **Route Application Server ID**: TLS security ID between provider platform and agent. The application server ID should be unique to the route address. This is an optional field. Leave empty to disable security.
-- **Route Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change the application server key if  the application server ID is not updated. This is an optional field. Leave empty to disable security.
+- **Application Server ID**: TLS security ID between provider platform and agent. The application server ID should be unique to the route address. This is an optional field. Leave empty to disable security.
+- **Application Server Key**: TLS security key between provider platform and agent. The value should be in hex and 16 bytes. Do not change the application server key if the application server ID is not updated. This is an optional field. Leave empty to disable security.
+- **Admin API version**: The version that the Thingpark admin API uses. The default is "latest".
+- **Core API version**:  The version that the Thingpark core API uses. The default is "latest".
```
Comment (manasesjesus) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:25]:
```suggestion
- **Admin API version**: The version that the ThingPark admin API uses. The default is "latest".
- **Core API version**:  The version that the ThingPark core API uses. The default is "latest".
```

**Review by ashhSAG93 — APPROVED** [review]

**Pull request merged**